tests/providers/google/cloud/hooks/test_dataprep.py

```diff
@@ -29,18 +31,29 @@
 
 class TestGoogleDataprepHook(TestCase):
     def setUp(self):
-        self.hook = dataprep.GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id")
+        with mock.patch(
+            "airflow.providers.google.cloud.hooks.dataprep.GoogleDataprepHook._token"
+        ) as token:
+            extra = {"token": token}
+
+        with mock.patch(
+            "airflow.providers.google.cloud.hook.dataprep.GoogleDataprepHook.get_connection"
+        ) as conn_mock:
+            conn_mock.return_value = Connection(
+                conn_id="dataprep_conn_id", extra=json.dumps(extra)
+            )
+            self.hook = dataprep.GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id")
```

Comment:
```suggestion
        with mock.patch(
            "airflow.providers.google.cloud.hook.dataprep.GoogleDataprepHook.get_connection"
        ) as conn_mock:
            conn_mock.return_value = Connection(
                conn_id="dataprep_conn_id", extra=json.dumps({"token": TOKEN})
            )
            self.hook = dataprep.GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id")
```
and make `TOKEN` a global constant that can be used in other tests

Reply:
We fixed this during call.